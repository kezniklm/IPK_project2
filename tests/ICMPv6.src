src MAC: 54:e1:ad:c1:2d:ee
dst MAC: 30:9c:23:28:c7:7a
frame length: 118 bytes
Internet Protocol Version 6
src IP: fe80::8f8a:681d:7918:7cab
dst IP: fe80::2d5:8ff8:38e:894
Internet Control Message Protocol version 6
ICMPv6 Echo Reply

0x0000:   30 9c 23 28 c7 7a 54 e1 ad c1 2d ee 86 dd 60 00   0.#(.zT...-...`.
0x0010:   c8 49 00 40 3a 40 fe 80 00 00 00 00 00 00 8f 8a   .I.@:@..........
0x0020:   68 1d 79 18 7c ab fe 80 00 00 00 00 00 00 02 d5   h.y.|...........
0x0030:   8f f8 03 8e 08 94 81 00 12 b1 00 28 00 07 56 d1   ...........(..V.
0x0040:   3a 64 00 00 00 00 8d 3f 05 00 00 00 00 00 10 11   :d.....?........
0x0050:   12 13 14 15 16 17 18 19 1a 1b 1c 1d 1e 1f 20 21   .............. !
0x0060:   22 23 24 25 26 27 28 29 2a 2b 2c 2d 2e 2f 30 31   "#$%&'()*+,-./01
0x0070:   32 33 34 35 36 37                                 234567
