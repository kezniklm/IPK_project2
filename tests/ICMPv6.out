timestamp: 2023-04-17T19:41:56.506+02:00
src MAC: 54:e1:ad:c1:2d:ee
dst MAC: 30:9c:23:28:c7:7a
frame length: 118 bytes
Internet Protocol Version 6
src IP: fe80::8f8a:681d:7918:7cab
dst IP: fe80::2d5:8ff8:38e:894
Internet Control Message Protocol version 6
ICMPv6 Echo Reply

0x0000:  30 9C 23 28 C7 7A 54 E1 AD C1 2D EE 86 DD 60 00         0.#(.zT...-...`.
0x0010:  C8 49 00 40 3A 40 FE 80 00 00 00 00 00 00 8F 8A         .I.@:@..........
0x0020:  68 1D 79 18 7C AB FE 80 00 00 00 00 00 00 02 D5         h.y.|...........
0x0030:  8F F8 03 8E 08 94 81 00 12 B1 00 28 00 07 56 D1         ...........(..V.
0x0040:  3A 64 00 00 00 00 8D 3F 05 00 00 00 00 00 10 11         :d.....?........
0x0050:  12 13 14 15 16 17 18 19 1A 1B 1C 1D 1E 1F 20 21         .............. !
0x0060:  22 23 24 25 26 27 28 29 2A 2B 2C 2D 2E 2F 30 31         "#$%&'()*+,-./01
0x0070:  32 33 34 35 36 37                                       234567
